src/common/constants.ts

```diff
@@ -45,6 +45,7 @@ export const constants = {
     autoLabelBatchSizeMax: 10,
     autoLabelBatchSizeMin: 3,
     showOriginLabelsByDefault: true,
+    enableMultiPageField: true,
```

Comment:
suggestion: default to `false` for backward compatibility, we will ask customer to change in their branch.